src_test/test_ppx_yojson.ml

```diff
@@ -220,6 +220,32 @@ let test_nostrict ctxt =
                (nostrict_of_yojson (`Assoc ["nostrict_field", (`Int 42);
                                             "some_other_field", (`Int 43)]))
 
+module Opentype =
+  struct
+    type 'a opentype = .. [@@deriving yojson]
+    type 'a opentype += A of 'a | B of string list [@@deriving yojson]
+  end
```

Comment:
Can you also add a signature to test that the signatures for externally visible modules are derived correctly?